src/main/java/com/hotels/beeju/HiveServer2JUnitRule.java

```diff
@@ -76,30 +76,22 @@ protected void init() throws Throwable {
     try (ServerSocket socket = new ServerSocket(0)) {
```

Comment:
I guess the code that's left here should move to some core class as well, maybe `HiveServer2Core`